modules/apps/saved-content/saved-content-service/src/main/java/com/liferay/saved/content/service/impl/SavedContentEntryLocalServiceImpl.java

```diff
@@ -103,6 +104,14 @@ public SavedContentEntry fetchSavedContentEntry(
 			classPK);
 	}
 
+	@Override
+	public List<SavedContentEntry> findSavedContentEntries(
+		long groupId, String className, long classPK) {
+
+		return savedContentEntryPersistence.findByG_C_C(
+			groupId, _classNameLocalService.getClassNameId(className), classPK);
+	}
```

Comment:
Why not create a `deleteSavedContentEntries` method instead? Build service should have created a `removeByG_C_` that you can call directly. This way, you avoid the need to loop in the model listener.

Reply:
great, thanks!